src/views/CloneRepo.vue

```diff
@@ -1,20 +1,110 @@
 <template>
   <ViewWrapper title="Clone down a repository">
-    <div>
+    <label for="repo-url-input">
+      <strong>
+        Enter url:
```

Comment:
Since this is such a specific field we can be more precise in the directions:
```suggestion
        Paste HTTPS or SSH URL to clone here:
```